framework/src/dm/arch/sidk_s5jt200/s5j_dm_connectivity.c

```diff
@@ -40,13 +40,25 @@
 #include <dm/dm_error.h>
 #include <dm/dm_connectivity.h>
 
+#include <apps/netutils/dhcpc.h>
 #include <apps/netutils/wifi/slsi_wifi_api.h>
+#include <apps/netutils/wifi/slsi_wifi_utils.h>
 #include <arpa/inet.h>
 
 #define DM_GET_INTERFACE_INFO 0
 #define DM_GET_IP_ADDR_INFO 1
 
 #define DM_CALLBACK_BUFFER_SIZE 10
+#define DM_BSSID_LEN            18
+#define DM_NET_DEVNAME          "wl1"
+
+/* Static data structure to hold results of a WiFi scan.
+ */
+static slsi_scan_info_t *wifiscan_result;
```

Comment:
g_wifi_scan_result & g_scan_iter(you can rename but make sure 'global_do_what') will be better